js/advanced-cam-modal.js

```diff
@@ -25,120 +25,43 @@ function typeofOperation(newval, objectseq) {
     cncVEngMode(objectseq);
     updateCamUserData(objectseq);
   } 
-
-
 }
 
 function initAdvancedCAM() {
   $('#statusBody2').on('keyup change', 'input, select', function() {
-    // console.log('Action')
-    var inputVal = $(this).val();
-    var newval = inputVal
+
     var id = $(this).attr('id');
-    // console.log(id)
     var objectseq = $(this).attr('objectseq');
-    // console.log('Value for ' +id+ ' changed to ' +newval+ ' for object ' +objectseq );
-    if (id.indexOf('tzstep') == 0) {
-      var numPass = Math.floor((parseFloat($('#tzdepth' + objectseq).val()) / parseFloat(newval)))
 
-      if ((parseFloat($('#tzdepth' + objectseq).val()) / parseFloat(newval)) - Math.floor(parseFloat($('#tzdepth' + objectseq).val()) / parseFloat(newval)) != 0) {
```

Comment:
I'm not familiar with what this is doing, but I assume it's ok to remove all of it?